src/Validatable.php

```diff
@@ -35,11 +35,7 @@ public function _getValidationData(): array
             return $this->getValidationData();
         }
 
-        if (method_exists($this, 'attributesToArray')) {
-            return $this->attributesToArray();
-        }
-
-        return [];
+        return $this->attributesToArray();
```

Comment:
👍 Yes, @robtownsend1, since this package assumes a `laravel model` context, this is a good default (since a model always has this method in the framework)